modules/network-observability-multitenancy.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-multi-tenancy{context}"]
+= Enable multi-tenancy in Network Observability
+Multi-tenancy in the Network Observability Operator allows and restricts individual user access, or group access, to the flows stored in Loki. Access is enabled for project admins. Project admins who have limited access to some namespaces can access flows for only those namespaces.
+
+.Prerequisite
+* You have installed link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7] 
+* The `FlowCollector` `spec.loki.authToken` configuration must be set to `FORWARD`.
```

Comment:
@Amoghrd I added the `FORWARD` requirement to the prerequisites. Let me know if you think it should be added somewhere else too.